add_me_to_your_PATH/sysimage.jl

```diff
@@ -18,11 +18,9 @@ Pkg.add("PackageCompiler")
 using PackageCompiler, UUIDs
 
 function main()
-    packages = [Symbol(k) for k in keys(project.dependencies)]
-    # exclude dependencies that are being developed
-    exclude = [Symbol(pi.name) for pi in values(Pkg.dependencies()) if !pi.is_tracking_registry]
+    packages = [Symbol(k) for k in keys(Pkg.project().dependencies)]
     # exclude some packages that can make sysimage creation or usage fail.
-    foreach(d -> push!(exclude, d), [:Revise, :PackageCompiler, :K8sClusterManagers])
+    exclude = [:Revise, :PackageCompiler, :K8sClusterManagers]
```

Comment:
Do we still need to exclude K8sClusterManagers?

Reply:
Not sure, haven't tried in a while, the new version might not have issues. Lemme give it a shot.

Reply:
I just tried with Revise and K8sClusterManagers both pinned / in the sysimage and it's working. Removing both from excludes.